lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -118,7 +119,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
             .subscribe((valuesToSetIfNotPresent) => {
                 const keys = Object.keys(valuesToSetIfNotPresent);
                 keys.forEach(key => {
```

Comment:
all this logic seems to be strange is here, are you sure it should not be all moved in the form model in the update method with all the other logic?

Reply:
Moved to the form model